<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyChoose</class>
 <widget class="QDialog" name="KeyChoose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выберите необходимый ключ</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Доступные ключи</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListView" name="listView"/>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KeyChoose</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KeyChoose</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
